pyproject.toml

```diff
@@ -45,7 +45,7 @@ dependencies = [
     "onnx>=1.16.0",
     "toolz>=0.12.1",
     "rich>=13.7.1",
-    "torchmetrics @ git+https://github.com/Lightning-AI/torchmetrics.git",
+    "torchmetrics>=1.4.2",
```

Comment:
should we maybe also update the `timm` version? Currently we point to main, but I think in the current release the features we need are available

Reply:
yeap was thinking the same..